tools/bundle-size-tools/src/utilities/getChunkParsedSize.ts

```diff
@@ -10,7 +10,7 @@ import { WebpackStatsJson } from '../BundleBuddyTypes';
  */
 export function getChunkParsedSize(stats: WebpackStatsJson, chunkId: string | number): number {
   if (stats.assets === undefined) {
-    throw new Error(`No assets property in the stats file, can't compute parsed sizes of chunks`);
+    throw new Error("No assets property in the stats file, can't compute parsed sizes of chunks");
```

Comment:
This looks like the only place we use double quotes. Something we need to tweak somewhere to keep it consistently single quotes?

Reply:
That's a good point. I think I should be able to include that in this PR.

Reply:
Ok I finally found out what's happening here. We have avoidEscape set to true in the root eslintrc.js which makes it so strings can use single or double quotes so long as the string contains a quote that would have to be escaped otherwise. Do we want to change this behavior?

Reply:
[Official docs](https://eslint.org/docs/2.0.0/rules/quotes) for more details

Reply:
@erinha friendly bump on this question :)

Reply:
Sorry I missed your reply! That makes sense, this is fine to leave as it is.